backend/src/main/java/com/jos/spotifyclone/controller/BrowseController.java

```diff
@@ -104,6 +106,24 @@ public FeaturedPlaylists getListOfFeaturedPlaylists() throws ParseException, Spo
         return spotifyConnect.getSpotifyApi().getListOfFeaturedPlaylists().build().execute();
     }
 
+    //http://localhost:8080/api/browse/getPlaylist?playlist_id=3AGOiaoRXMSjswCLtuNqv5
+    @GetMapping("/getPlaylist")
+    public Playlist getPlaylist(@RequestParam String playlist_id) throws ParseException, SpotifyWebApiException, IOException {
```

Comment:
Should these not live in the playlist controller?

Reply:
i included them here since theres get tracks get albums etc and i thought id put these here too,should i move?